### DISCUSSION TITLE
Raven sent a message that was not shown immediately

### DISCUSSION THREAD

**Discussion by alishirini** [The-Commit-Company/raven#1170] [Q&A]
Dear guys,

I am new to Raven.

I installed Frappe 15 and everything is good but I have a problem with immediately showing the enter a message, it's not showing and I must close and open chat after that to show the message!

Maybe need config Webhook on frappe or other things?

Please help

Best Regards,

**Comment by nikkothari22** [comment]
Hey @alishirini 

It looks like your Websockets are not working. 

Can you tell me more about the setup? I am assuming this is self-hosted. 

Also, can you go to "System Health Report" in the Desk interface and check if the Socket.io ping is working or not.

**Reply by alishirini** [reply]
@nikkothari22 Hi 

Yes, Self-hosting and all apps and modules work (LMS, HR, ...)

I attached my errors in "System Health Report"


![Screenshot 2024-12-02 110645](https://github.com/user-attachments/assets/3d4b2585-b904-4aa4-a97a-14776b5cb4f1)
![Screenshot 2024-12-02 110651](https://github.com/user-attachments/assets/cef9dc95-33a2-4949-9c20-c1fc1d1ed120)
![Screenshot 2024-12-02 110708](https://github.com/user-attachments/assets/0e47ab4a-44a5-41db-816f-a5a65fdb45fc)

**Reply by alishirini** [reply]
What should I do?

**Reply by alishirini** [reply]
sample result of my host:

curl "http://xxxxxx:9000/socket.io/?EIO=4&transport=polling"

`0{"sid":"szAOG4nOR5b8w2c4AAA3","upgrades":["websocket"],"pingInterval":25000,"pingTimeout":20000,"maxPayload":1000000}`

**Reply by nikkothari22** [reply]
I think you'll need to fix your Socket.io connection. This is a Frappe installation issue. 

I don't know your exact setup (if it's production or development, or on HTTP/HTTPS). I would suggest you to search for socketIO related issues on the Frappe forum.
